tests/test_mdlp.py

```diff
@@ -32,165 +34,52 @@ def test_find_cut_no_cut():
-        [ 0.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 2.,  1.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 2.,  1.,  1.,  2.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  2.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 2.,  0.,  2.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 2.,  1.,  1.,  1.],
-        [ 2.,  1.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 2.,  1.,  2.,  2.],
-        [ 1.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 0.,  0.,  1.,  1.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 1.,  0.,  2.,  2.],
-        [ 1.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  1.],
-        [ 2.,  1.,  2.,  2.],
-        [ 1.,  0.,  1.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  1.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  1.,  2.],
-        [ 2.,  0.,  1.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  1.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  1.],
-        [ 2.,  0.,  2.,  1.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  1.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 1.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.]]
-
-    assert_array_equal(transformed, expected,
-                       err_msg="MDLP output is inconsistent with previous runs.")
+@pytest.mark.skip(reason='demonstrating behavior to be fixed')
```

Comment:
For now, let's defer this to a later PR.

Reply:
👍